<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="buttonColor1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonColor2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPenWidth">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxPenStyle">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBack">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonForward">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCounter">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxFigure">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCapStyle">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonZ2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonZ1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonZ3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionArrow"/>
   <addaction name="actionPencil"/>
   <addaction name="actionFigure"/>
   <addaction name="actionExcrete"/>
   <addaction name="actionPepette"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFigure">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/toolBar/figure.png</normaloff>:/images/toolBar/figure.png</iconset>
   </property>
   <property name="text">
    <string>figure</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/toolBar/pencil.png</normaloff>:/images/toolBar/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/toolBar/cursor_drag_hand.png</normaloff>:/images/toolBar/cursor_drag_hand.png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionExcrete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/toolBar/excrete.png</normaloff>:/images/toolBar/excrete.png</iconset>
   </property>
   <property name="text">
    <string>Excrete</string>
   </property>
  </action>
  <action name="actionPepette">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/toolBar/pipette.png</normaloff>:/images/toolBar/pipette.png</iconset>
   </property>
   <property name="text">
    <string>Pepette</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
